content_zh/_index.md

```diff
@@ -59,15 +59,15 @@ title: Istio
 </script>
 
 <main class="landing">
-    <div class="banner">
+    <div id="banner">
         {{< inline_image "landing/istio-logo.svg" >}}
-        <div class="hero-text">
-            <h1 class="hero-label">Istio</h1>
-            <h1 class=“hero-lead”>用于连接、保护、控制和观测服务。
+        <div id="hero-text">
+            <h1 id="hero-label">Istio</h1>
+            <h1 id=“hero-lead”>用于连接、保护、控制和观测服务。
```

Comment:
Change the quote from `“` to `"`.

Reply:
Huh, I wonder how those snuck in there...

Reply:
Because some IME will auto-complete the punctuation and replace it with the Chinese quota.